frontend/src/App.tsx

```diff
@@ -49,6 +54,21 @@ export default function App () {
     // Applies custom theming from public/theming.js
     applyTheme(window.custom_theme);
 
+    
+    useEffect(() => {
+        dispatch(getConfiguration('global'));
+    }, [dispatch]);
+
+    useEffect(() => {
+        if (configLoadError) {
+            notificationService.generateNotification(
+                'Error loading app configuration. Restrictive default policy has been applied in its place. Consult with system admin to resolve issue.',
```

Comment:
where do we actually set the the restrictive configuration specified here: `Restrictive default policy has been applied in its place.`